java/data-management/strings-i/replace-a-substring.md

```diff
@@ -26,3 +30,23 @@ car = car.replace(oldColor, newColor);
 ```
 
 In the above example, `replace` searches the string `car` for the value of `oldColor`, which is "blue". Any instances of "blue" are then replaced with `newColor`, which is "red". The final value of `car` is "red car".
+
+---
+
+## Revision
+
+Complete the following code to change your opinion:
+
+```java
+String myOpinion = "I love running in cold weather";
+String oldOpinion = "love";
+String newOpinion = "hate";
+??? = myOpinion.replace(
+      oldOpinion,???
```

Comment:
formatting feels off here?

Reply:
I think @Stefan-Stojanovic did this so that it would fit in the screen perfectly.

Reply:
Sure thing, but in that case it should be like this:

```suggestion
      oldOpinion, ???
```

or like this

```suggestion
      oldOpinion,
      ???
```